ansible/utils/install_lxc_packages.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- hosts: localhost
+  become: true
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
```

Comment:
do we need to specify this?
It may vary in the customer's env.

Reply:
Yes, we need to specify the interpreter.
As per my knowledge, we don't see the issue with it until python3 or a higher version is available on the ansible node.